test/functional/feature_loan_vault.py

```diff
@@ -348,10 +347,6 @@ def run_test(self):
 
         self.nodes[0].generate(1)
         self.sync_blocks()
-        vault1 = self.nodes[0].getvault(vaultId1)
-        assert_equal(vault1['loanAmount'], ['0.50000000@TSLA'])
-        assert_equal(vault1['collateralValue'], Decimal(2.00000000))
-        assert_greater_than(vault1['loanValue'],Decimal(0.5))
```

Comment:
Why remove but not update the value check?

Reply:
Sure! forgot to update! will do now